### ISSUE TITLE
HTML needs default language identified with `lang` attribute

### ISSUE THREAD

**Issue by jendowns** [carbon-design-system/gatsby-theme-carbon#557]
## Detailed description

The `HTML` element does not have a default `lang` set. In the DAP tool, it generates the following violation:

![Screen Shot 2019-11-22 at 2 27 16 PM](https://user-images.githubusercontent.com/9057921/69458330-ac654580-0d34-11ea-956f-3a79c9766b77.png)

### What WCAG 2.1 checkpoint does the issue violate?

http://www.ibm.com/able/guidelines/ci162/language_of_page.html

## Steps to reproduce the issue

Run the DAP tool on any page using `gatsby-theme-carbon`, and you'll see the this violation.

**Comment by jendowns** [comment]
I can propose a fix for this shortly 👍
